### PR TITLE
Calendar was created

### PR DESCRIPTION
**Pull request by NazariiAlieksieiev**
- [DEMO LINK](https://NazariiAlieksieiev.github.io/layout_calendar/)
- [TEST REPORT LINK](https://NazariiAlieksieiev.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,68 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start--day--tue calendar__month--lenght--28">
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
```
Comment (ivanchuk-yurii) [inline src/index.html]:
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  width: getCalendarWidht($cell-size);
+    
+  &__day {
+    @include square($cell-size);
+    background-color: #eee;
+    border: 1px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/calendar.scss]: move this to `html`

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    font-size: 30px;
+    transition: transform 0.5s;
+
+    &:hover {
+      background-color: pink;
+      cursor: pointer;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/calendar.scss]: should be in `&__day`

File: src/styles/blocks/calendar.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    &__day {
+      &:nth-child(#{$index}) {
+        &::before {
+          content: "#{$index}";
+        }
+      }
+    }
+  }
+
+  $days:
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/calendar.scss]: move this to `variables.scss`

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/styles/utils/_resets.scss [hunk 5]
```diff
@@ -5,3 +5,10 @@ body {
 * {
   box-sizing: border-box;
 }
+
+html {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (ivanchuk-yurii) [inline src/styles/utils/_resets.scss:13]: I meant to move just `font-family`, not `flex`)

Comment (ivanchuk-yurii) [inline src/styles/utils/_resets.scss:10]: I meant to move just `font-family`, not `flex`)

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/styles/utils/_resets.scss (hunk 5 above)
Comment (NazariiAlieksieiev) [reply src/styles/utils/_resets.scss:10]: done

**Comment by OlEzhA0** [comment]
Screenshot 2022-12-11 at 9 54 08 PM
tests report link unavailable

File: src/styles/blocks/calendar.scss [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      &:nth-child(#{$index}) {
+        &::before {
+          content: "#{$index}";
+        }
+      }
+    }
+  }
+
+  @each $name, $day in $days {
+    &__start--day--#{$name} :first-child {
```
Comment (OlEzhA0) [inline src/styles/blocks/calendar.scss:37]:
```suggestion
    &__start--day--#{$name}:first-child {
```

File: src/styles/blocks/calendar.scss [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  }
+ 
+  @for $index from $first-day through $month-lenght {
+    &__day {
+      &:nth-child(#{$index}) {
+        &::before {
+          content: "#{$index}";
+        }
+      }
+    }
```
Comment (OlEzhA0) [inline src/styles/blocks/calendar.scss:33]:
```suggestion
    &__day:nth-child::before {
      content: "#{$index}";
    }
```

**Comment by NazariiAlieksieiev** [comment]
image

**Comment by NazariiAlieksieiev** [comment]
> Screenshot 2022-12-11 at 9 54 08 PM
> 
> tests report link unavailable 

In my phone its works

**Review by OlEzhA0 — APPROVED** [review]
Hey! good job
I added few small comments, you can check it

**Comment by NazariiAlieksieiev** [comment]
> Hey! good job
> 
> I added few small comments, you can check it

Thanks)

**Pull request closed without merging**